src/index.js

```diff
@@ -39,15 +39,15 @@ module.exports = Class.extend({
          return;
       }
 
-      /**
-       * Each entry in the this._pendingAssociations array looks like this:
-       * {
-       *    "fnLogicalName": "YourFnNameLambdaFunction",
-       *    "distLogicalName": "WebsiteDistribution",
-       *    "fnCurrentVersionOutputName": "YourFnNameLambdaFunctionQualifiedArn",
-       *    "eventType": "origin-request",
-       * }
-       */
+        /**
```

Comment:
I'm a little confused why the intention is changing here. This shouldn't need to change for the PR to pass the eslint rules.